cli.js

```diff
@@ -63,7 +65,7 @@ global.applications = [];
 
 // Execute shell command displaying output and possible errors
 const execute = (cmd, callback) => {
-  cp.exec(cmd, { cwd: __dirname }, (error, stdout /* stderr */) => {
+  cp.exec(cmd, { cwd: __dirname }, (error, stdout) => {
```

Comment:
Not printing `stderr` here may not be that good of an idea since all the info about any errors that occur during `cmd` execution will be in the `stderr` here.

I once had a situation where I had to debug something in this package, and all I had to do to get enough information about the issue was to modify this function to just print `stderr`.